calicoctl/commands/node/run.go

```diff
@@ -256,6 +274,58 @@ Description:
 		fmt.Println(err)
 		os.Exit(1)
 	}
+
+	// Create the command to follow the docker logs for the calico/node
+	logCmd := exec.Command("docker", "logs", "calico-node", "--follow")
+
+	// Get the stdout pipe
+	outPipe, err := logCmd.StdoutPipe()
+	if err != nil {
+		fmt.Println("Error querying calico/node logs:")
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	// Start following the logs.
+	err = logCmd.Start()
+	if err != nil {
+		fmt.Println("Error querying calico/node logs:")
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	// Read stdout until the node fails, or until we see the output
+	// indicating success.
+	stdout := ""
+	txt := make([]byte, 1000)
```

Comment:
might be better to use the `bufio` package here to create a read buffer since `outPipe` implements the `ReadCloser` interface `bufReader := bufio.NewReader(outPipe)` and `ReadLine()` that way you don't have to create a buffer of 1000 size which might not fit the need, and won't have to do `txt[:n]`